### PR TITLE
Db connection form

### PR DESCRIPTION
**Pull request by Rahul83326**
Layout Changes

### CODE REVIEW

File: silzila-frontend/src/Components/DataConnection/DataSetup.css [hunk 1]
```diff
@@ -252,9 +243,12 @@ Data Connections
...
 	width: 22rem;
 	display: flex;
-	padding: 5px 20px 5px 30px;
+	padding: 5px 8px 5px 12px;
 	border: 1px solid rgba(224, 224, 224, 1);
 	color: rgb(101, 101, 101);
+	white-space: nowrap; /* Prevent text wrapping */
+    overflow: hidden; /* Hide overflowing text */
+    text-overflow: ellipsis; /* Display ellipsis (...) for overflowed text */
 }
```
Comment (silzila) [inline silzila-frontend/src/Components/DataConnection/DataSetup.css:252]: Don't change the alignment.

File: silzila-frontend/src/Components/DataConnection/DataSetup.css [hunk 2]
```diff
@@ -453,29 +453,50 @@ DataHome
...
-	display: flex;
-	position: fixed;
-    bottom: 1px;
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    height: 3.5rem;
+	z-index: 1;
+    background: rgb(255, 255, 255); /* Fully opaque background color */
+    border-top: 2px solid rgba(224, 224, 224, 1);
```
Comment (silzila) [inline silzila-frontend/src/Components/DataConnection/DataSetup.css:480]: Here also: give proper indentation.

**Review by silzila — COMMENTED** [review]
if the formatting or indentation is correct, then don't change it.

File: silzila-frontend/src/Components/DataConnection/DataSetup.css (hunk 2 above)
Comment (Rahul83326) [reply silzila-frontend/src/Components/DataConnection/DataSetup.css:480]: Ok Sir

**Pull request merged**